code/modules/mob/living/simple_animal/hostile/mimic.dm

```diff
@@ -441,6 +446,14 @@ var/global/list/item_mimic_disguises = list(
 	copied_object = meat_type //Without this line, mimics would spawn items they're disguised as. Since they're relatively weak and can appear as gatling guns, this is required!
 	..(gibbed)
 
+/mob/living/simple_animal/hostile/mimic/crate/item/attackby(obj/W, mob/user)
+	if(angry) //If we're angry - proceed as normal
+		return ..()
+	else if(istype(W, /obj/item/stack/package_wrap))
+		return attack_hand(user) //Less chill than crate mimics
```

Comment:
this else tree is very confusing because you get the same outcome for both the else if and the else